.github/workflows/core-smart-contract-deploy.yml

```
@@ -79,9 +79,7 @@ jobs:
           RPC_URL: ${{ secrets.RPC_URL }}
           DEPLOYER_PRIVATE_KEY: ${{ secrets.DEPLOYER_PRIVATE_KEY }}
           DATASET_REGISTRY_ADDRESS: ${{ vars.DATASET_REGISTRY_ADDRESS }}
-          # For Verification
-          EXPLORER_API_KEY: ${{ secrets.EXPLORER_API_KEY }}
-          IS_VERIFICATION_API_V2: ${{ vars.IS_VERIFICATION_API_V2 }}
+          HARDHAT_IGNITION_CONFIRM_DEPLOYMENT: ${{ vars.HARDHAT_IGNITION_CONFIRM_DEPLOYMENT }}
```

Too much config, this should simply be:
```suggestion
          HARDHAT_IGNITION_CONFIRM_DEPLOYMENT: false
```